app/javascript/app/components/ndcs-map/ndcs-map-selectors.js

```diff
@@ -31,16 +31,21 @@ export const getIndicatorsParsed = createSelector(
 );
 
 export const getSelectedCategory = createSelector(
-  [state => state.categorySelected, getCategories],
-  (selected, categories = []) => {
+  [state => state.categorySelected, getCategories, getIndicatorsData],
+  (selected, categories = [], indicators) => {
     if (categories.length > 0) {
       if (selected) {
         const filtered = categories.filter(
```

Comment:
can we avoid ambiguous names for variables if we are mixing indicators and categories in one selector. 'categoriesFiltered' or similar.